website/content/2016-05-31-looking-for-guest-posts.md

```diff
@@ -16,7 +16,7 @@ Details of how to guest post on here:
 1. This site is [a Jekyll site on GitHub](https://github.com/HypothesisWorks/HypothesisWorks.github.io).
    To add a post, create a markdown file in the _posts directory with the
    appropriate structure and send a pull request.
-2. You will want to add an entry for yourself to [the authors data file](https://github.com/HypothesisWorks/HypothesisWorks.github.io/blob/master/_data/authors.yml)
+2. You will want to add an entry for yourself to [the authors data file](https://github.com/HypothesisWorks/HypothesisWorks.github.io/blob/main/_data/authors.yml)
```

Comment:
for (much) later: ooh, on top of the general finish-the-website-update https://github.com/HypothesisWorks/hypothesis/issues/2943, I should make sure that all the urls point to this repo and archive the old one.